package.json

```diff
@@ -5,9 +5,9 @@
   "repository": "git@github.com:plone/plone.staticresources.git",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "mockup": "https://github.com/plone/mockup.git",
+    "jquery": "https://github.com/jquery/jquery-dist.git#1.11.3",
     "less": "https://github.com/less/less.git#v2.7.3",
-    "requirejs": "~2.2.0",
-    "jquery": "https://github.com/jquery/jquery-dist.git#1.11.3"
+    "mockup": "https://github.com/plone/mockup.git#momentjsLazyLoadLocales",
```

Comment:
We need first to have this mockup branch build properly and after it is merged to master we can use it here

Reply:
Phantom tests fail because Karma can't serve, for instance `moment-url/en.js` (imported in `mockup/patterns/moment/pattern.js`) since the definition of `moment-url` lives in `plone.staticresources/plone/staticresources/profiles/default/resources.xml`, which mockup isn't aware of:

```
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR: 'There is no timestamp for /node_modules/moment/locale/pt/pt.js!'
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR: 'There is no timestamp for /node_modules/moment/locale/pt/pt.js!'

PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR LOG: 'There is no timestamp for /node_modules/moment/locale/pt/pt.js!'
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  {
    "message": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18",
    "str": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18"
  }
PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  {
    "message": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18",
    "str": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18"
  }

PhantomJS 2.1.1 (Mac OS X 0.0.0) ERROR
  {
    "message": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18",
    "str": "Error: Script error for \"moment-url/pt\", needed by: mockup-patterns-moment\nhttp://requirejs.org/docs/errors.html#scripterror\nat node_modules/requirejs/require.js:143:18"
  }
PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.459 secs / 0 secs)


PhantomJS 2.1.1 (Mac OS X 0.0.0): Executed 0 of 0 ERROR (0.459 secs / 0 secs)
```

We tried to update mockup/config.js to declare `moment-url` (following the lead of `mockup-ui-url` among many others `-url` suffixed):

```
'moment-url': 'node_modules/moment/locale',
```

But it was not enough: Karma still 404s for http://0.0.0.0:9876/base/node_modules/moment/locale/en.js (while moment is correctly served at http://0.0.0.0:9876/base/node_modules/moment/moment.js?f5ff9e6fdcddb2951851412bc021015b6b59054c)

Reply:
@ale-rt, tests fixed in `mockup`, and also merged to master. Could you please review again so we can run CI again?